docs/install_guides/configurations-for-ncs2.md

```diff
@@ -11,68 +11,30 @@
 
 Once you have OpenVINO™ Runtime installed, follow these steps to be able to work on NCS2:
 
-1. Go to the install_dependencies directory:
-   ```sh
-   cd <INSTALL_DIR>/install_dependencies/
-   ```
-2. Run the `install_NCS_udev_rules.sh` script:
-   ```
-   ./install_NCS_udev_rules.sh
-   ```
-3. You may need to reboot your machine for this to take effect.
-
-You've completed all required configuration steps to perform inference on Intel® Neural Compute Stick 2. 
-Proceed to the [Get Started Guide](@ref get_started) section to learn the basic OpenVINO™ workflow and run code samples and demo applications.
-
-@sphinxdirective
-
-.. _ncs guide raspbianos:
-
-@endsphinxdirective
-
-## Raspbian OS
```

Comment:
Just want to confirm: are we sure to remove configurations for Raspbian OS? Why?

Reply:
Because it will be the same to the general Linux* and we don't need to split general instruction to specific one.